src/main/web/templates/handlebars/partials/json-ld/publisher.handlebars

```diff
@@ -0,0 +1,5 @@
+"publisher": {
+    "@type": "Organization",
```

Comment:
Out of scope but this should prob be: `GovernmentOrganization` https://schema.org/GovernmentOrganization

Reply:
👍